.github/workflows/tests.yaml

```diff
@@ -137,6 +142,10 @@ jobs:
     - name: ${{ matrix.test.name }}
       run: ${{ matrix.test.command }}
       env: ${{ matrix.test.env }}
+    - name: Print stderr logs
+      run: grep -H "" *.stderr.log || true
+    - name: Error if stderr logs are not empty
```

Comment:
Maybe a bit extreme to fail the test? IDK, just a personal feeling, it's useful to have feedback on any new stderr message, maybe we can add a `continue-on-error: true` so it's visible but doesn't outright fail the test if the overall test suite is successful.

Reply:
That's the idea to catch immediately a warning, so we are not surprised later.

Then if we don't plan on fixing the warning, we can filter it out from the logs ([38badb7](https://github.com/scality/utapi/pull/1318/commits/38badb76a0402ff224b46f0b9b85bca94e9f3273))

A `continue-on-error: true` could be added at any moment if there are warnings that can't be fixed to pass the tests, but ideally I'd like to fail to make sure we notice it. 

And I'd like to have this in other components as well in long term